### ISSUE TITLE
Importing multiple environments does not import all selected files

### ISSUE THREAD

**Issue by ebonnot** [a85/POSTMan-Chrome-Extension#4260]
## App Details:
```
Postman for Linux
Version 6.0.7
linux 4.4.0-116-generic / x64
```

## Issue Report:
Since version 6, I am unable to import 4 environment files (only 2 are imported). 
Works fine on version 5.5.2

1. go to "Manage environments"
2. click on "Import" button 
3. select 4 files,
4. validate

Only 2 files are imported even though messages for 4 files successfully imported appear.

![screenshot at 2018-03-01 12 19 54](https://user-images.githubusercontent.com/23345518/36842348-b652fb12-1d4b-11e8-9029-6cf99f13d875.png)

**Comment by thecuriousdev** [comment]
Hey, @ebonnot thanks for reporting this issue. I'm investigating this issue, will keep you posted here.

**Comment by thecuriousdev** [comment]
@ebonnot thanks for your patience. The issue got fixed in 6.0.9, you will get the update soon. Please let me know if it occurs again after the update.

**Comment by ebonnot** [comment]
@thecuriousdev : it's ok in 6.0.9. Thanks !
